src/semantic/src/site/globals/site.overrides

```diff
@@ -289,17 +291,50 @@ h2 {
   margin: 0rem 0.5rem 1rem 0.5rem;
 }
 
+.change-address-link {
+  margin: -0.5rem 0rem 0.7rem 0rem !important;
+  font-size: 12px !important;
+}
+
+#user-input-address-grid {
+  width: 100% !important;
+  margin: 0 !important;
+}
+
+#user-input-address-row {
+  border-bottom: thin solid gray !important;
+  padding: 0rem !important;
+  padding-left: .2rem!important;
+  padding-right: .2rem!important;
+  margin: 0 .5rem 1rem .5rem;
+}
+
+#user-input-address-column {
+  padding-left: 0rem !important;
+}
+
+#adress-icon-column {
+  padding: 0.2rem 0rem 0.2rem 0.2rem !important;
+}
+
+#userInputAddress {
```

Comment:
please stick to `kebab-case` in css

Reply:
The "issue" is that we are passing in the id in the `onChangeHandler` function in our CreateImageEntry component - when setting the state for the value in the form, and in JS it should be camel case right.. Since we are trying to manipulate the input field with CSS we needed too choose between camel case and kebab-case for this id. But I guess we can check and see how we can use other ways to manipulate the styling, OR set the state without passing in the id..

Reply:
No time to check. This stays as is. Refactor after vacation!